frontend/package.json

```diff
@@ -21,23 +21,23 @@
     "build:analyze": "export _ODH_OUTPUT_ONLY=true; run-s build build:bundle-profile build:bundle-analyze",
     "build:bundle-profile": "webpack --config ./config/webpack.prod.js --profile --json > ./bundle.stats.json",
     "build:bundle-analyze": "webpack-bundle-analyzer ./bundle.stats.json",
-    "build:storybook": "build-storybook -s ./config",
+    "build:storybook": "storybook build -s ./config",
     "build:clean": "rimraf ./public",
     "build:prod": "webpack --config ./config/webpack.prod.js",
     "format": "prettier --write \"./**/*.ts\" \"./**/*.tsx\" \"./**/*.scss\"",
     "start:dev": "webpack serve --hot --color --config ./config/webpack.dev.js",
-    "test": "run-s test:lint test:jest test:type-check",
-    "test:jest": "jest",
-    "test:ci": "export CI=true; jest --collectCoverage",
-    "test:clearCache": "jest --clearCache",
-    "test:dev": "run-s test:lint test:local",
+    "test": "run-s test:lint test:unit test:type-check test:integration-ci",
+    "test:dev": "run-s test:lint test:unit test:accessibility test:integration",
     "test:fix": "eslint --ext .json,.js,.ts,.jsx,.tsx ./src --fix",
     "test:lint": "eslint --max-warnings 0 --ext .json,.js,.ts,.jsx,.tsx ./src",
-    "test:local": "jest --watchAll",
     "test:type-check": "tsc --noEmit",
-    "test:storybook": "test-storybook --verbose",
-    "test:storybook-ci": "npx concurrently -k -s first -n \"SB,TEST\" -c \"magenta,blue\" \"npm run build:storybook --quiet && npx http-server storybook-static --port 6006 --silent\" \"npx wait-on tcp:127.0.0.1:6006 && npm run test:storybook\"",
-    "storybook": "start-storybook -p 6006 -s ./config"
+    "test:jest": "jest",
+    "test:unit": "npm run test:jest",
+    "test:accessibility": "test-storybook --verbose",
+    "test:integration": "playwright test ./src/__tests__/integration",
+    "test:integration-ci": "npx concurrently -k -s first -n \"SB,TEST\" \"npm run build:storybook --quiet && npx http-server storybook-static --port 6006 --silent\" \"npx wait-on tcp:127.0.0.1:6006 && npm run test:accessibility && npm run test:integration\"",
+    "test:e2e": "export E2E=1; playwright test ./src/__tests__/e2e",
```

Comment:
This does not work for me -- what am I missing?

I have client and server running, I run this and I always get a failure on:
```
 Received: {"_frame": {"_guid": "frame@6f12dfd3c3b8a82344ecf9dba74d88fa", "_type": "Frame"}, "_selector": "internal:text=\"Error creating project\"i"}

      33 |   await page.getByLabel('Name *', { exact: true }).fill('e2e-test-project');
      34 |   await page.getByRole('button', { name: 'Create' }).click();
    > 35 |   await expect(page.getByText('Error creating project')).toBeFalsy();
         |                                                          ^
      36 |   await expect(page.locator('tr', { has: page.locator('text="e2e-test-project"') })).toBeTruthy();
      37 |   await page.getByRole('link', { name: 'Data Science Projects', exact: true }).click();
      38 |

        at /Users/andrewballantyne/code/others/odh-dashboard/frontend/src/__tests__/e2e/CreateEditDeleteProject.spec.ts:35:58
```

Reply:
i will look into it

Reply:
e2e has been fixed. there was issues with the expect calls